easybuild/easyconfigs/c/CrayGNU/CrayGNU-19.08.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'CrayToolchain'
+
+name = 'CrayGNU'
+version = '19.08'
+
+homepage = 'https://pubs.cray.com/discover'
+description = """Toolchain using Cray compiler wrapper, using PrgEnv-gnu module
+(PE release: August 2019).\n"""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+dependencies = [
+    # PrgEnv version is not pinned, as Cray recommends to use the latest
+    # (default) version
+    ('PrgEnv-gnu', EXTERNAL_MODULE),
+    ('atp/2.1.3', EXTERNAL_MODULE),
+    # gcc versions later than 8 are not supported by cuda/10.1:
+    # -----------------   gcc/4   gcc/5   gcc/6   gcc/7   gcc/8   gcc/9
+    # cuda/09.2 < gcc/8   Y       Y       Y       Y       -       -
+    # cuda/10.0 < gcc/8   Y       Y       Y       Y       -       -
+    # cuda/10.1 < gcc/9   Y       Y       Y       Y       Y       -
+    # -----------------
+    ('gcc/8.3.0', EXTERNAL_MODULE),
+    ('modules/3.2.11.2', EXTERNAL_MODULE),
```

Comment:
Please remove this line (see comment above).

Reply:
Done.